src/ts/components/seekbarcontrols.ts

```diff
@@ -0,0 +1,108 @@
+import { UIUtils } from '../uiutils';
+import { DOM } from '../dom';
+import { PlayerAPI } from 'bitmovin-player';
+import { VolumeController } from '../volumecontroller';
+
```

Comment:
Why isn't this a `class SeekBarController` to fit the remainder of the architecture?